lib/prediction_class.py

```diff
@@ -132,6 +132,12 @@ def prediction_classes(cls):
     def prediction_methods(cls):
         return sorted([prediction_class.__name__ for prediction_class in cls.prediction_classes()])
 
+    @classmethod
+    def prediction_methods_with_all(cls):
+        methods = cls.prediction_methods()
+        methods.extend(['all', 'all_mhc_i', 'all_mhc_ii'])
```

Comment:
In run_utils.py, it's looking for `all_class_i` and `all_class_ii`.  Do these get converted from `all_mhc_i` and `all_mhc_ii` somehow?

Reply:
my vote is for "all_class_i" over "all_mhc_i", fwiw